eng/common/scripts/TypeSpec-Project-Generate.ps1

```diff
@@ -6,83 +6,16 @@ param (
     [ValidateNotNullOrEmpty()]
     [string] $ProjectDirectory,
```

Comment:
I just want to ensure you are aware of the new tsp-client tool that is eventually going to replace this script. See https://github.com/Azure/azure-sdk-tools/pull/6843

Reply:
Thanks, I've started talking to @catalinaperalta about the features we need supported in the tsp-client tools and removed most of the changes to the Typespec-Project-Generate and npm scripts for this PR.